mkt/extensions/models.py

```diff
@@ -145,6 +145,13 @@ def get_fallback(cls):
     def get_indexer(self):
         return ExtensionIndexer
 
+    def is_dummy_content_for_qa(self):
+        """
+        Returns whether this extension is a dummy extension used for testing
```

Comment:
Yeah, `mkt.search.utils.extract_popularity_trending_boost()` uses that. I'll modify the comment.